packages/babel-helper-module-transforms/src/rewrite-live-references.ts

```diff
@@ -1,7 +1,5 @@
-import assert from "assert";
 import { template, types as t } from "@babel/core";
 import type { NodePath, Visitor, Scope } from "@babel/core";
-import simplifyAccess from "@babel/helper-simple-access";
```

Comment:
Can you remove this from `package.json`?

Also (in a follow up PR) I'd like for this package to be inlined inside `plugin-transform-modules-commonjs`, since now it's only used there.

Reply:
Sure! I would try removing it first since it introduces an extra traversal.